content/en/blog/2019/istioctl-describe/index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "'istioctl describe' in Istio 1.3"
+description: Introducing a command-line utility to find configuration affecting pod traffic.
+publishdate: 2019-09-20
```

Comment:
@esnible: Why is this a release blocker if it won't be published until September 20? Please confirm that this is a blocker for Istio 1.3

Reply:
It is not a blocker.  We can publish it when it is ready.